android/src/org/ppsspp/ppsspp/NativeActivity.java

```diff
@@ -681,7 +681,7 @@ public void onConfigurationChanged(Configuration newConfig) {
 			Point sz = new Point();
 			getDesiredBackbufferSize(sz);
 			if (sz.x > 0) {
-				mGLSurfaceView.getHolder().setFixedSize(sz.x/2, sz.y/2);
+				mGLSurfaceView.getHolder().setFixedSize(sz.x, sz.y);
```

Comment:
Hmm, yeah.  It's been there [for a while](https://github.com/hrydgard/ppsspp/commit/1012f46dee3473376eff2a549c64a7f2356461b0#diff-b466153019b453972ac2157a51b625ebR468).

-[Unknown]